src/scripts/Outland/nagrand.cpp

```diff
@@ -41,6 +41,26 @@ npc_corki2
 go_corkis_prison3
 npc_corki3
 npc_kurenai_captive
+npc_gunthock
```

Comment:
They are all already implemented in the file there was just no reference at the top of the file. For example, I didn't think the warmual pyre was implemented at all because I didn't see it in the original list. It wasn't until I did a search for the quest id later that I found it was implemented. I figured I would update the list so people can see at a glance in the future and avoid the issue I had, if that is not the purpose of the list I can remove them.